src/ddmd/parse.d

```diff
@@ -3134,21 +3133,21 @@ final class Parser : Lexer
         case TOKinterface:
             if (!id)
                 error(loc, "anonymous interfaces not allowed");
-            a = new InterfaceDeclaration(loc, id, baseclasses);
+            a = astFam.iInterfaceDeclaration(loc, id, baseclasses);
```

Comment:
So this would become `a = new ASTFamily.InterfaceDeclaration(loc, id, baseclasses);`.